beater/server.go

```diff
@@ -89,6 +96,10 @@ func newServer(logger *logp.Logger, cfg *config.Config, tracer *apm.Tracer, repo
 	if err != nil {
 		return server{}, err
 	}
+	grpcServer, err := newGRPCServer(logger, cfg, tracer, reporter, httpServer.TLSConfig)
+	if err != nil {
+		return server{}, err
+	}
 	jaegerServer, err := jaeger.NewServer(logger, cfg, tracer, reporter)
```

Comment:
Could we add a comment here and/or in the `jaeger` package that this is deprecated? Might be confusing for other devs to come accross the muxed gRPC server and the deprecated separate grpc server logic.